jetstream_pt/ray_engine.py

```diff
@@ -76,6 +82,65 @@ def prefill(
     # the worker itself manages and maintains the prefill states.
     return None
 
+  # pylint: disable-next=all
+  def disaggregated_prefill(
```

Comment:
Nit: this function doesn't seem quite different from interleave_prefill either, consider consolidate in the future if not now.

Reply:
Sounds good, I may have extra change on disaggregated function, will consider to merge it in future if possible.

Reply:
Changed,  I removed the interleave and disaggregated prefill function. Both interleave and disaggregated call prefill function now,